Library/Formula/foremost.rb

```diff
@@ -11,7 +11,12 @@ def install
       s.change_make_var! "RAW_FLAGS", ENV.cflags
     end
 
-    system "make", "mac"
+    if OS.mac?
+      system "make", "mac"
+    else
+      inreplace "config.c", "/usr/local/etc/", "#{etc}/"
```

Comment:
This line isn't strictly necessary, but it makes the formula respect Linuxbrew conventions better. (And it can't be changed by altering a Makefile variable: it is hard-coded into the source).

Without it, on startup the command tries to look for the default config file in `/usr/local` (which by sheer coincidence is where upstream Homebrew would have put it).

Technically speaking, it *still* throws a warning message on startup because of the rename to `foremost.conf.default`, but given that the formula already did that I assume that was by design.